Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+FROM sociomantictsunami/dlang:xenial-v3
```

Comment:
This should also be just `:v3` and `DIST` should be defined as an env variable, all in the separate stage.